packages/RdfService/index.ts

```diff
@@ -842,7 +842,15 @@ export class RdfService {
   */
   async runQuery<T>(query: string): Promise<QueryResponse<T>> {
     if (isEmptyString(query)) throw Error("runQuery: A valid query is required");
-    const response = await fetch(this.queryEndpoint + encodeURIComponent(this.sparqlPrefixes + query), { 'headers': { 'Expects': 'application/sparql-results+json' } })
+    const response = await fetch(this.queryEndpoint, {
+      method: 'POST',
+      headers: {
+        'Expects': 'application/sparql-results+json',
+        'Content-Type': 'application/x-www-form-urlencoded',
+        'Accept': 'application/sparql-results+json',
+      },
+      body: new URLSearchParams({ query })
```

Comment:
**RE**: `x-www-form-urlencoded'`
Oh, is this how fueski works?

Reply:
All else being equal (Which it may NOT be equal, I'm naive)

This would probs be more conventional
```
const response = await fetch(this.queryEndpoint, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'Accept': 'application/sparql-results+json',
  },
  body: JSON.stringify({ query })
});
```

I'm not sure about the `Expects` header also - but I'm checking docs

Reply:
Yea, it turned out the content type should actually be `application/sparql-query` and I removed `Expects` and it worked fine. I was curious about that one my self but left it in because I didn't want to inadvertently break anything.